src/mui/field/DateField.js

```diff
@@ -4,6 +4,7 @@ import get from 'lodash.get';
 const DateField = ({ source, record, showTime = false, elStyle }) => {
     if (!record) return null;
     const value = get(record, source);
+    if (value == null) return null;
```

Comment:
We don't do it anywhere else. And I don't see what you'd want to display if there is no value?